### PR TITLE
Consolidate docids for breaking changes

### PR DESCRIPTION
**Pull request by twsouthwick**
Now that docids can refer to groupings (types/namespaces/etc), this consolidates
the breaking change docids to simplify the listing

### CODE REVIEW

**Comment by twsouthwick** [comment]
@mjrousos

File: docs/BreakingChanges/107 Accessing a WPF DataGrid's selected items from a handler of the DataGrid's UnloadingRow event can cause a NullReferenceException.md [hunk 1]
```diff
@@ -24,7 +24,6 @@ This issue has been fixed in the .NET Framework 4.6 and may be addressed by upgr
 ### Affected APIs
 * `E:System.Windows.Controls.DataGrid.UnloadingRow`
 * `E:System.Windows.Controls.DataGrid.UnloadingRowDetails`
-* `E:System.Windows.Controls.DataGrid.UnloadingRowGroup`
```
Comment (twsouthwick) [inline docs/BreakingChanges/107 Accessing a WPF DataGrid's selected items from a handler of the DataGrid's UnloadingRow event can cause a NullReferenceException.md]: Some of these APIs we listed are actually only in Silverlight or .NET Core, so are not ones we want to flag for breaking changes

File: docs/BreakingChanges/075 TargetFrameworkName for default app domain no longer defaults to null if not set.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ The TargetFrameworkName was previously null in the default app domain, unless it
 Code should be updated to not depend on `AppDomainSetup.TargetFrameworkName` defaulting to null. If it is required that this property continue to evaluate to null, it can be explicitly set to that value.
 
 ### Affected APIs
-* `M:System.AppDomainSetup.get_TargetFrameworkName`
+* `P:System.AppDomainSetup.TargetFrameworkName`
```
Comment (mjrousos) [inline docs/BreakingChanges/075 TargetFrameworkName for default app domain no longer defaults to null if not set.md]: This change does specifically affect the getter (and not setter), so should it maybe be left with the more narrow accessor doc id?

File: docs/BreakingChanges/119 ADO.NET now attempts to automatically reconnect broken SQL connections.md [hunk 3]
```diff
@@ -19,11 +19,11 @@ Beginning in the .NET Framework 4.5.1, the .NET Framework will attempt to automa
...
-* `M:System.Data.IDbConnection.set_ConnectionString(System.String)`
-* `M:System.Data.SqlClient.SqlConnection.set_ConnectionString(System.String)`
-* `M:System.Configuration.ConnectionStringSettings.set_ConnectionString(System.String)`
-* `M:System.Data.Common.DbConnection.set_ConnectionString(System.String)`
-* `M:System.Data.Common.DbConnectionStringBuilder.set_ConnectionString(System.String)`
+* `P:System.Data.IDbConnection.ConnectionString`
+* `P:System.Data.SqlClient.SqlConnection.ConnectionString`
+* `P:System.Configuration.ConnectionStringSettings.ConnectionString`
+* `P:System.Data.Common.DbConnection.ConnectionString`
+* `P:System.Data.Common.DbConnectionStringBuilder.ConnectionString`
```
Comment (mjrousos) [inline docs/BreakingChanges/119 ADO.NET now attempts to automatically reconnect broken SQL connections.md]: Similar to my last comment, these breaks are specifically in how setting the connection string is handled, so it may be better to only list the set methods.

File: docs/BreakingChanges/131 PreviewLostKeyboardFocus is called repeatedly if its handler shows a message box.md [hunk 4]
```diff
@@ -24,10 +24,10 @@ There are two options to work around this issue -
...
 
 ### Affected APIs
-* `M:System.Windows.ContentElement.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
-* `M:System.Windows.IInputElement.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
-* `M:System.Windows.UIElement.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
-* `M:System.Windows.UIElement3D.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
+* `E:System.Windows.ContentElement.PreviewLostKeyboardFocus`
+* `E:System.Windows.IInputElement.PreviewLostKeyboardFocus`
+* `E:System.Windows.UIElement.PreviewLostKeyboardFocus`
+* `E:System.Windows.UIElement3D.PreviewLostKeyboardFocus`
```
Comment (mjrousos) [inline docs/BreakingChanges/131 PreviewLostKeyboardFocus is called repeatedly if its handler shows a message box.md]: Flagging the event will alert people who invoke the event in addition to just those who attach handlers. This is a tricky issue to detect, so neither of those is perfect but I think only flagging cases when a handler is attached might be better scoped.

File: docs/BreakingChanges/137 Only Tls 1.0 1.1 and 1.2 protocols supported in System.Net.ServicePointManager and System.Net.Security.SslStream.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ Starting with the .NET Framework 4.6, the `System.Net.ServicePointManager` and `
 ### Recommended Action
 The recommended mitigation is to upgrade the sever-side app to Tls1.0, Tls1.1, or Tls1.2. If this is not feasible, or if client apps are broken, the AppContext class can be used to opt out of this feature in either of two ways: 
 
-1. By programmatically setting compat switches on the AppContext, as explained [here](TBD)
+1. By programmatically setting compat switches on the AppContext, as explained [here](http://blogs.msdn.com/b/dotnet/archive/2015/04/29/net-announcements-at-build-2015.aspx#dotnet46)
```
Comment (mjrousos) [inline docs/BreakingChanges/137 Only Tls 1.0 1.1 and 1.2 protocols supported in System.Net.ServicePointManager and System.Net.Security.SslStream.md]: Thanks!

File: docs/BreakingChanges/142 WCF Binding with the TransportWithMessageCredential security mode.md [hunk 6]
```diff
@@ -30,7 +30,6 @@ Because this is an opt-in feature, it should not affect the behavior of existing
 ### Affected APIs
 * `F:System.ServiceModel.BasicHttpSecurityMode.TransportWithMessageCredential`
 * `F:System.ServiceModel.BasicHttpsSecurityMode.TransportWithMessageCredential`
-* `F:System.ServiceModel.PollingDuplexHttpSecurityMode.TransportWithMessageCredential`
```
Comment (mjrousos) [inline docs/BreakingChanges/142 WCF Binding with the TransportWithMessageCredential security mode.md]: Why was this removed?

File: docs/BreakingChanges/146 CurrentCulture flows across Tasks.md [hunk 7]
```diff
@@ -24,10 +24,8 @@ AppContext.SetSwitch("Switch.System.Globalization.NoAsyncCurrentCulture", true);
 ```
 
 ### Affected APIs
-* `M:System.Globalization.CultureInfo.set_CurrentCulture(System.Globalization.CultureInfo)`
```
Comment (mjrousos) [inline docs/BreakingChanges/146 CurrentCulture flows across Tasks.md]: Again, I'm not sure that we need to make this change because I think users who only get the CurrentCulture are less likely to care about this.

File: docs/BreakingChanges/075 TargetFrameworkName for default app domain no longer defaults to null if not set.md (hunk 2 above)
Comment (twsouthwick) [inline docs/BreakingChanges/075 TargetFrameworkName for default app domain no longer defaults to null if not set.md]: Hmm... the msdn generation only works for properties, not specific methods. I can try to add a filter to find the property when given a setter.

File: docs/BreakingChanges/131 PreviewLostKeyboardFocus is called repeatedly if its handler shows a message box.md (hunk 4 above)
Comment (twsouthwick) [inline docs/BreakingChanges/131 PreviewLostKeyboardFocus is called repeatedly if its handler shows a message box.md]: Same as with properties, MSDN documentation only works with events, not the add/remove methods. I'll look to see if I can do a conversion when creating the documentation

File: docs/BreakingChanges/142 WCF Binding with the TransportWithMessageCredential security mode.md (hunk 6 above)
Comment (twsouthwick) [inline docs/BreakingChanges/142 WCF Binding with the TransportWithMessageCredential security mode.md]: This is only in Silverlight, so it won't ever be found in .NET framework desktop apps

Comment (mjrousos) [inline docs/BreakingChanges/142 WCF Binding with the TransportWithMessageCredential security mode.md]: Got it

File: docs/BreakingChanges/075 TargetFrameworkName for default app domain no longer defaults to null if not set.md (hunk 2 above)
Comment (mjrousos) [inline docs/BreakingChanges/075 TargetFrameworkName for default app domain no longer defaults to null if not set.md]: Ah, I hadn't considered these from the perspective of auto-generating MSDN docs. Yea, if we can map accessors to properties for purposes of doc generation, that would be ideal. If not, making them properties (and adding a little bit of noise to reports) is probably acceptable in the name of getting docs auto-generated.

**Comment by twsouthwick** [comment]
@mjrousos Reverted the changes that expanded to whole properties/events when only one of the methods is affected

**Comment by mjrousos** [comment]
LGTM
